uplc-semantics.md

```diff
@@ -35,9 +36,15 @@ module UPLC-SEMANTICS
        <env> RHO => .Map </env>
   requires X in_keys(RHO)
 
+  rule <k> X:UplcId => #gLookup(X) ... </k>
+       <env> RHO => .Map </env>
+  requires #inKeysgEnv(X)
+   andBool notBool(X in_keys(RHO))
+
   rule <k> X:UplcId => (error) ... </k>
        <env> RHO </env>
-  requires notBool(X in_keys(RHO))
+  requires notBool(#inKeysgEnv(X))
+   andBool notBool(X in_keys(RHO))
```

Comment:
I'm not sure it's a good idea to hardcode the global environment into the interpreter. I guess it's kind of harmless, since it can be overridden in RHO. But I have reservations about adding behavior to the semantics that aren't in the specification.